package.json

```diff
@@ -127,7 +127,7 @@
     "jetifier": "1.6.4",
     "metro-react-native-babel-preset": "0.56.0",
     "mini-css-extract-plugin": "^0.9.0",
-    "node-sass": "4.14.1",
+    "node-sass": "^4.14.1",
     "resolve-url-loader": "^3.1.1",
```

Comment:
```suggestion
```

Reply:
Isn't this used at all?

Reply:
@mradkov Saw the different diff. But maybe we don't need `resolve-url-loade` at all. I need to check that. I added it because of aliases and we don't use them in sass right now.

Reply:
@mradkov just saw some wrong diff. it used in config. But I'm not sure that we need that package at all it for the situation when `since you will want to co-locate your assets with your .scss partials.`

Reply:
We use it inside of scss loader

Reply:
@DeeJayElly yes, but until we don't need to co-locate your assets with your .scss partials we don't need that package and loader. At this time all assets are located outside of scss directories.

Reply:
@DeeJayElly also jitsi upstream repo don't use it. And I just added it because I thought that it can help to resolve the webpack aliases, but it's for the different thing and also we don't use  aliases.